apps/condo/domains/resident/schema/RegisterResidentService.js

```diff
@@ -48,7 +48,7 @@ const RegisterResidentService = new GQLCustomSchema('RegisterResidentService', {
                 const [property] = await PropertyAPI.getAll(
                     context,
                     { address_i: propertyAddress, deletedAt: null },
-                    { sortBy: ['createdAt_ASC'], first: 1 }
+                    { sortBy: ['isApproved_ASC', 'createdAt_ASC'], first: 1 }
```

Comment:
we need to test it

Reply:
Probably, we need to use the manageResidentToPropertyAndOrganizationConnections here?

Reply:
Yes, there are a test for this:

`connects property with matched address to resident, prefers isApproved == true`


```(js)
 it('connects property with matched address to resident, prefers isApproved == true', async () => {
        const adminClient = await makeLoggedInAdminClient()

        const [organization] = await registerNewOrganization(adminClient)
        const [OlderButApprovedProperty] = await createTestProperty(adminClient, organization, {
            map: buildingMapJson,
            isApproved: true,
        })

        const [organization2] = await registerNewOrganization(adminClient)
        await createTestProperty(adminClient, organization2, {
            map: buildingMapJson,
            address: OlderButApprovedProperty.address,
            addressMeta: OlderButApprovedProperty.addressMeta,
        })

        const payload = {
            address: OlderButApprovedProperty.address,
            addressMeta: OlderButApprovedProperty.addressMeta,
        }

        const [obj, attrs] = await registerResidentByTestClient(adminClient, payload)
        expect(obj.id).toMatch(UUID_RE)
        expect(obj.dv).toEqual(1)
        expect(obj.sender).toEqual(attrs.sender)
        expect(obj.v).toEqual(1)
        expect(obj.address).toEqual(attrs.address)
        expect(obj.addressMeta).toStrictEqual(attrs.addressMeta)
        expect(obj.user.id).toEqual(adminClient.user.id)
        expect(obj.property.id).toEqual(OlderButApprovedProperty.id)
        expect(obj.organization.id).toEqual(organization.id)
        expect(obj.unitType).toEqual(FLAT_UNIT_TYPE)
    })
```